src/config.json

```diff
@@ -62,6 +62,7 @@
     },
     "tokens": []
   },
+  "RADICLE_HTTP_API":"",
```

Comment:
Let's call this `radicle: { api: string }`, so it matches the format of the safe config.

Reply:
Okay yeah no problem, I kept it that way to maintain the `process.env` style for a eventual return to injected env vars

Reply:
Ah I see, we can just do `config.radicle.api = import.meta.env.RADICLE_HTTP_API` when that happens